### ISSUE TITLE
 Login error, server busy. Waiting 10 seconds to try again.

### ISSUE THREAD

**Issue by jrdncchr** [PokemonGoF/PokemonGo-Bot#3209]
Please check configuration at http://jsonlint.com/ before posting an issue.

### Expected Behavior
Login successful.

### Actual Behavior
 Login error, server busy. Waiting 10 seconds to try again.

### Steps to Reproduce
python pokecli.py

### Other Information
OS: Windows 10
Git Commit: 3c9806b3de71b5c2c38ba92f22ed662901ee700d
Python Version: Python 2.7.10

![aaa](https://cloud.githubusercontent.com/assets/3001348/17504683/b972745a-5e2d-11e6-8c3e-2f7e1d2cca0c.PNG)

**Comment by shivamkalra** [comment]
So did you wait 10 seconds?

**Comment by jrdncchr** [comment]
Of course its been repeating, and cannot log in.

**Comment by ponytails** [comment]
try to use app password instead of your email/user password

**Comment by jrdncchr** [comment]
When I go to my google account apppasswords, its empty.

**Comment by innrwrld** [comment]
I get similar behavior on macOS today, ever since the mobile app updates yesterday evening.

```
2016-08-09 08:37:41,868 [PokemonGoBot] [INFO] [login_started] Login procedure started.
Traceback (most recent call last): ...

pgoapi.exceptions.NotLoggedInException
2016-08-09 08:37:42,015 [sentry.errors] [ERROR] Sentry responded with an error: 'ascii' codec can't decode byte 0x9c in position 1: ordinal not in range(128) (url: https://app.getsentry.com/api/90254/store/) ...

UnicodeDecodeError: 'ascii' codec can't decode byte 0x9c in position 1: ordinal not in range(128)
2016-08-09 08:37:42,017 [sentry.errors.uncaught] [ERROR] [u'NotLoggedInException', u'  File "pokecli.py", line 521, in <module>', u'  File "pokecli.py", line 137, in main', u'  File "pokecli.py", line 146, in report_summary', u'  File "pokemongo_bot/metrics.py", line 76, in capture_stats', u'  File "pokemongo_bot/api_wrapper.py", line 90, in call', u'  File "pokemongo_bot/api_wrapper.py", line 58, in can_call']
```

Which result from:

```
2016-08-09 08:34:53,578 [PokemonGoBot] [INFO] [login_started] Login procedure started.
2016-08-09 08:34:54,434 [PokemonGoBot] [INFO] [login_failed] Login error, server busy. Waiting 10 seconds to try again.
2016-08-09 08:35:05,825 [PokemonGoBot] [INFO] [login_failed] Login error, server busy. Waiting 10 seconds to try again.
^C2016-08-09 08:35:06,498 [PokemonGoBot] [INFO] [bot_exit] Exiting bot.
2016-08-09 08:35:09,954 [pokemongo_bot.api_wrapper] [WARNING] Server seems to be busy or offline - try again - 4/15
```

I've tested this against both PTC & Google account types.

**Comment by evan0322** [comment]
@innrwrld Could you share how did you solve the issue? I got the same error here.

**Comment by innrwrld** [comment]
@evan0322 Mine was related to firewall restrictions on a corporate network.  I forgot we have SSL decryption going on & that caused my connections to fail, plus since PMG stuff is "games" related it gets APP filtered by a different type of firewall ruleset, so I got a double whammy.  Once I remembered I had to do something else to resolve, it was all good.  Not sure however what may be causing your woes, unless of course you're attempting to also run from a corporate environment.
